client-app/src/desktop/tabs/home/widgets/activity/ActivityWidget.ts

```diff
@@ -46,3 +59,14 @@ const bbar = hoistCmp.factory({
         });
     }
 });
+
+const gitHubErrorPanel = hoistCmp.factory(({error}) => {
+    return errorMessage({
+        omit: !error,
+        error: error,
+        title: 'Error loading GitHub panel',
+        message: error?.['isRoutine']
```

Comment:
use the error type to dictate what to display to the user. If routine, we can feel free to expose the message content, otherwise provide a generic message